continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -1,3 +1,2 @@
 python:
-    - 3.8
```

Comment:
I'm not sure why only 3.8 and 3.9 are here. cc @charlesbluca

Reply:
Think this file is just a remnant of when we were doing architectured builds of Distributed; now that we're building `noarch` don't think this file should have any impact and should be safe to remove altogether.

cc @jakirkham in case I've missed something here

Reply:
Yeah would just drop this file